svgdigitizer/svgdigitizer.py

```diff
@@ -157,20 +158,20 @@ def paths(self):
         the `<path>` tags that are not used for other purposes such as pointing
         to axis labels.
         """
-        return {
-            path.getAttribute('id'): self.parse_pathstring(path.getAttribute('d'))
-            for path in self.doc.getElementsByTagName("path")
-            # Only take paths into account which are not in groups since those are
-            # the paths pointing to labels on the axes.
-            if path.parentNode.nodeName != 'g' or
-                # This is complicated by the fact that layers as created by inkscape
-                # are also groups. Such layers have the 'inkscape:groupmode' set.
-                path.parentNode.getAttribute("inkscape:groupmode") == 'layer' or
-                # But this attribute goes away when exporting to plain SVG. In
-                # such case, we recover layers as the groups that contain more
-                # than one path.
-                len(path.parentNode.getElementsByTagName("path")) > 1
-        }
+        paths = {}
```

Comment:
Have we now decided that there can be no `<path>` unless it is grouped with some text? If so, we could probably refactor things a bit here by introducing a method that returns all the pairs `(text string, path)` for all the paths in the SVG (and warns about paths that are not following this scheme.)
This function could then be used everywhere and simplify the code that searches the groupings.

Reply:
Yes, I think we did. @DunklesArchipel